st2common/st2common/util/jinja.py

```diff
@@ -60,12 +60,15 @@ def get_filters():
     from st2common.jinja.filters import time
     from st2common.jinja.filters import version
     from st2common.jinja.filters import json_escape
+    from st2common.jinja.filters import jmespath_query
 
     # IMPORTANT NOTE - these filters were recently duplicated in st2mistral so that
     # they are also available in Mistral workflows. Please ensure any additions you
     # make here are also made there so that feature parity is maintained.
     return {
         'decrypt_kv': crypto.decrypt_kv,
+        'from_json_string': data.from_json_string,
```

Comment:
We actually already have pack actions for that, but I guess I still see some value in having those operations available as filters as well.

Having said that, we need to be careful down the road and come up with some rules / conventions on what goes in a filter and what in an action.

Reply:
Makes sense.

My use case for `from_json_string` is reading values out of the datastore. Commonly in our `action` metadata files we have default values set to Jinja expressions that read from datastore and contain JSON strings. In order for us to utilize this data effectively we need to parse the JSON out into objects. 

Part of this problem stems from the fact that the datastore only stores information in `string` format, so you're forced to serialize on either end. If typed loading/unloading was supported, then the need for this filter (in our use case) would go away.

Reply:
On a related note - we actually have action for that which handles serialization / de-serialization (`st2kv.get_object` `st2kv.set_object` or something along those lines) and could maybe also work for some of your use cases.